gst/tensor_filter/tensor_filter_tensorflow_lite_core.cc

```diff
@@ -328,9 +327,11 @@ TFLiteCore::invoke (const GstTensorMemory * input, GstTensorMemory * output)
 
   for (int i = 0; i < outputTensorMeta.num_tensors; i++) {
     if (outputTensorMeta.info[i].type == _NNS_FLOAT32) {
-      output[i].data = interpreter->typed_output_tensor < float >(i);
+      memcpy (output[i].data, interpreter->typed_output_tensor < float >(i),
```

Comment:
Note that `memcpy` usage is related to issue #513. 
(Using the memcpy is discussed  at the week meeting on Oc-16-2018.)

Reply:
Don't do this. Consider it as a binary blob without any types.